### PR TITLE
[RLlib] Fix lstm use prev action reward

### PR DESCRIPTION
**Pull request by sven1977**
This PR fixes the currently broken `lstm_use_prev_action_reward` flag for default lstm models (`model.use_lstm=True`).

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

File: rllib/utils/tracking_dict.py [hunk 1]
```diff
@@ -17,6 +17,13 @@ def __init__(self, *args, **kwargs):
     def set_get_interceptor(self, fn):
         self.get_interceptor = fn
 
+    def copy(self):
```
Comment (sven1977) [inline rllib/utils/tracking_dict.py:20]: This was a bug.
When copied in `ModelV2.forward()`, all tracking-dict properties (_lazy_tensor_dict) are gone and we are left with a plain dict.

File: rllib/models/torch/recurrent_net.py [hunk 2]
```diff
@@ -123,6 +130,18 @@ def forward(self, input_dict, state, seq_lens):
         # Push obs through "unwrapped" net's `forward()` first.
         wrapped_out, _ = self._wrapped_forward(input_dict, [], None)
 
+        # Concat. prev-action/reward if required.
```
Comment (sven1977) [inline rllib/models/torch/recurrent_net.py:133]: This logic was missing since our move to ModelV2.

File: rllib/models/tf/recurrent_net.py [hunk 3]
```diff
@@ -151,6 +158,20 @@ def forward(self, input_dict, state, seq_lens):
         # Push obs through "unwrapped" net's `forward()` first.
         wrapped_out, _ = self._wrapped_forward(input_dict, [], None)
 
+        # Concat. prev-action/reward if required.
```
Comment (sven1977) [inline rllib/models/tf/recurrent_net.py:161]: This logic was missing since our move to ModelV2.

File: rllib/agents/ppo/tests/test.py [hunk 4]
```diff
@@ -1,43 +0,0 @@
-import unittest
```
Comment (sven1977) [inline rllib/agents/ppo/tests/test.py:1]: None of these functions were needed anywhere. Removed helper functions and their tests entirely.

**Review by ericl — APPROVED** [review]

**Pull request merged**